src/Controls/tests/UITests/Tests/Issues/Issue18753.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18753 : _IssuesUITest
+	{
+		public Issue18753(TestDevice device) : base(device) { }
+
+		public override string Issue => "[D8] Editor IsTextPredictionEnabled works";
+
+		[Test]
+		public async Task Issue18753Test()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Mac, TestDevice.Windows },
+				"Currently IsKeyboardShown is not implemented.");
+
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android },
+				"Currently is failing.");
+
+			App.WaitForElement("WaitForStubControl");
+
+			// 1. Enter the string 'hi my name is james' exactly (case-sensitive).
+			App.EnterText("EditorIsTextPredictionDisabled", "hi my name is james");
+			// 2. The test fails if the keyboard presents word suggestions.
+
+			// Focus the Editor.
+			App.Click("EditorIsTextPredictionDisabled");
+
+			if (App.IsKeyboardShown())
+			{
+				await Task.Delay(1000);
+				VerifyScreenshot("Issue18753IsTextPredictionDisabled");
+				App.DismissKeyboard();
+			}
+
+			// 3. Enter the string 'hi my name is james' exactly (case-sensitive).
+			App.EnterText("EditorIsTextPredictionEnabled", "hi my name is james");
+			// 4. The test fails if the keyboard does not present word suggestions.
+
+			// Focus the Editor.
+			App.Click("EditorIsTextPredictionEnabled");
+
+			if (App.IsKeyboardShown())
+			{
+				await Task.Delay(1000);
+				VerifyScreenshot("Issue18753IsTextPredictionEnabled");
```

Comment:
Not sure the test is testing for text suggestions. There is a spell check error, but in the screenshot there are no suggestions. I wonder if you should be stopping mid way through a work and see if "hi my nam" suggestes the word "name"?

Also, the if block here will pass if there is no keyboard, but we _want_ the keyboard, right? Should no keyboard rather be a fail or inconclusive since we cannot determine if it passed since no suggestions were actually presented.